src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -106,7 +106,24 @@ <h1 class="subheader__title">
 
                     <div class="subheader__teaser">
                         {% placeholder "course_teaser" or %}
-                        <div class="empty">{% trans 'Add a video or teaser.' %}</div>
+                            {% get_placeholder_plugins "course_cover" as plugins or %}
+                                {% if current_page.publisher_is_draft %}
+                                    <p class="empty">{% trans 'Add a teaser video or add a cover image below and it will be used as teaser image as well.' %}</p>
+                                {% endif %}
```

Comment:
@sampaccoud @jbpenrath I changed this to display edit message only in draft page, and I added according test.

Reply:
Good idea!